### PR TITLE
Portfolio accessibility

### PR DESCRIPTION
**Pull request by bertinkambale**
We evaluate our website against accessibility best practices and implement any necessary changes to ensure that our website is accessible
-Page titles: **no issues found**
-Image text alternatives** (alt text)
-Text headings: replace h5 with h2 tag
-Color contrast: Add more contrast to the text
-Resize: **no issues found**
-Interaction: **no issues found**
-Moving content: **no issues found**
-Multimedia: **no issues found**
-The basic structure of the page: **no issues found**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -91,9 +91,9 @@ <h1>Multi-Post<br />Stories</h1>
         <div class="new-style">
           <h2>Canopy</h2>
           <img src="images_for_second_section/Counter.svg" alt="" />
-          <h3>Back End Dev</h3>
+          <h2>Back End Dev</h2>
           <img src="images_for_second_section/Counter.svg" alt="" />
-          <h3>2015</h3>
+          <h2>2015</h2>
```
Comment (anasbassi) [inline index.html:96]: replaced h3 tags with h2 tags

File: style.css [hunk 2]
```diff
@@ -173,7 +173,7 @@ body {
 }
 
 .ancrotag li {
-  color: #6070ff;
+  color: #3e50f0;
```
Comment (anasbassi) [inline style.css:176]: Add more contras to text

**Review by anasbassi — COMMENTED** [review]
In this project, we performed an accessibility test on the website, which resulted in adding contrast to some text, adding the alt attribute to some images, and replacing some h3 and h5 tags with an h2 tag.

File: index.html [hunk 3]
```diff
@@ -51,7 +51,7 @@ <h1>
           you see and have a project you need coded, don’t hestiate to contact
           me.
         </p>
-        <h3>LET’S CONNECT</h3>
+        <h2>LET’S CONNECT</h2>
         <ul>
           <li><img src="Icons/twitter.svg" alt="" /></li>
           <li><img src="Icons/imagelinkd.svg" alt="" /></li>
```
Comment (tresorsawasawa) [inline index.html:57]: - [ ] Kindly make sure every image has `alt` with appropriate alternative text, that's useful for interacting or understanding the page has an alt text. Images with `alt` are helpful for people who are blind and use screen readers can hear the alt text read out, and people who have turned off images to speed download or save bandwidth can see the alt text. Make sure you apply for all images in your project to make them more accessible.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @bertinkambale  and @anasbassi 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Whoistolu — APPROVED** [review]
### Hi Team
@bertinkambale and @anasbassi 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**